stginga/nbinteract.py

```diff
@@ -15,10 +15,44 @@
 from ginga.AstroImage import AstroImage
 from ginga.web.pgw import Widgets, js, PgHelp, ipg
 
-__all__ = ['start_server', 'GingaServer', '']
+__all__ = ['GingaServer']
```

Comment:
yep, `start_server` is a holdover from an earlier way I had this structured.  So the change you made is correct.